modules/virt-using-virt-must-gather.adoc

```diff
@@ -19,10 +19,10 @@ The default data collection includes information about the following resources:
 
 .Procedure
 
-* Run the following command to collect data about {VirtProductName}:
+* Run the following command to collect data about {VirtProductName}
 +
 [source,terminal,subs="attributes+"]
 ----
 $ oc adm must-gather --image-stream=openshift/must-gather \
```

Comment:
--image-stream=openshift/must-gather thsi is not required.

Reply:
Fixed! :)